interface/forms/newpatient/common.php

```diff
@@ -562,10 +562,19 @@ function cancelClickedOld() {
                                 <label for='facility_id' class="text-right"><?php echo xlt('Facility'); ?>:</label>
                                 <select name='facility_id' id='facility_id' class='form-control' onChange="getPOS()" <?php echo ($mode === "followup") ? 'disabled' : ''; ?> >
                                     <?php
+                                    $care_team_facility = null;
+                                    if (!empty($GLOBALS['set_service_facility_encounter'])) {
+                                        $care_team_facility = sqlStatement("SELECT `care_team_facility` FROM `patient_data` WHERE `pid` = ?", array($_SESSION['pid']));
```

Comment:
Would use sqlQuery here instead which will return in an array.
then below pertinent lines would do:
```
&& !empty($care_team_facility['care_team_facility'])
```

```
$def_facility = $care_team_facility['care_team_facility'];
```